src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -163,6 +167,20 @@ struct miopen_convolution
         auto w_desc                = make_tensor(reshape_if_1d(inputs[1]), int8_x4_format);
         auto y_desc                = make_tensor(reshape_if_1d(output_shape));
         std::size_t workspace_size = 0;
+
+        // TODO:  Using an environment variable to disable convolution tuning while still allowing
+        // other (i.e. GEMM) tuning with --exhaustive-tune, is a workaround for a problem in which
+        // convolution tuning takes multiple hours.
+        const bool convo_tune_enabled = enabled(MIGRAPHX_ENABLE_CONV_TUNING{});
+        if(ctx.get_exhaustive_tune_flag() and not convo_tune_enabled)
+        {
+            std::cerr
+                << "WARNING: MIGraphX convolution tuning not enabled. To run tuning, set both the "
```

Comment:
This causes a flood of warning messages. Do we have a tidy way to limit this message to only output once per compilation?

Reply:
This should be removed, we want to enable miopen tuning by default with the exhaustive flag. If the user only wants to tune miopen then they can set the find enforce env variables.

Reply:
I've refactored this so that instead of specially DISabling tuning for convolutions, user can specially ENable it for GEMMs by setting MIGRAPHX_ENABLE_GEMM_TUNING.  See  src/targets/gpu/include/migraphx/gpu/gemm.hpp:142